applications/callflow/src/cf_util.erl

```diff
@@ -339,7 +339,16 @@ vm_count_by_owner(<<_/binary>> = AccountDb, <<_/binary>> = OwnerId) ->
 %%--------------------------------------------------------------------
 -spec alpha_to_dialpad(ne_binary()) -> ne_binary().
 alpha_to_dialpad(Value) ->
-    << <<(dialpad_digit(C))>> || <<C>> <= strip_nonalpha(wh_util:to_lower_binary(Value))>>.
+    Stripped = [strip_nonalpha(C) || <<C>> <= wh_util:to_lower_binary(Value)],
```

Comment:
You are technically correct, the best kind of correct.  I have submitted an updated version.